packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -215,6 +215,10 @@ export interface ExpoConfig {
        */
       keyid?: string;
     };
+    /**
+     * Headers for `requestHeadersKey`
```

Comment:
```suggestion
     * Extra HTTP headers to include in HTTP requests made by expo-updates. These may override preset headers.
```
(from expo-updates readme: https://github.com/expo/expo/tree/main/packages/expo-updates)